packages/server/lib/config.ts

```diff
@@ -1,15 +1,29 @@
 import _ from 'lodash'
-import type { ResolvedFromConfig } from '@packages/types'
 import * as configUtils from '@packages/config'
 
 export const setUrls = configUtils.setUrls
 
-export function getResolvedRuntimeConfig (config, runtimeConfig) {
```

Comment:
Removed this as it's no longer needed if we're dropping `resolved` from the API call